packages/fxa-customs-server/test/local/dataflow_tests.js

```diff
@@ -1,179 +1,768 @@
 /* Any copyright is dedicated to the Public Domain.
  * http://creativecommons.org/publicdomain/zero/1.0/ */
 
+const { assert } = require('chai')
 const proxyquire = require('proxyquire')
 const sinon = require('sinon')
-const { test } = require('tap')
+const { test: tapTest } = require('tap')
 
 let log
 let config
 let dataflow
 let PubSub
 let pubsub
+let sandbox
 let subscription
 
-test(
-  'shared setup', async (t) => {
-    log =  {
-      error: sinon.spy(),
-      info: sinon.spy()
-    }
+function reset () {
+  if (sandbox) {
+    sandbox.reset()
+  }
 
-    subscription = {
-      on: sinon.spy()
+  config = {
+    dataflow: {
+      enabled: true,
+      gcpPubSub: {
+        projectId: 'foo',
+        subscriptionName: 'bar'
+      }
     }
+  }
+}
+
+function test(testName, testFunction) {
```

Comment:
Tap is one of the reasons working in custom-server is a little more difficult, this makes it slightly more tolerable 👍🏽. But yea, this reminds me that we should eventually standardized with other repos and use mocha.